packages/next/src/client/components/react-dev-overlay/_experimental/internal/helpers/clsx.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * "Utility for constructing className strings conditionally."
+ * @reference https://github.com/lukeed/clsx/blob/925494cf31bcd97d3337aacd34e659e80cae7fe2/src/lite.js
+ */
+export function clsx(...args: (string | undefined | null | false)[]): string {
+  var i = 0,
```

Comment:
it can just be `args.filter(Boolean).join(' ')`, let's avoid inline other public source code, it could have issue without fully inlining the code